tf2onnx/graph.py

```diff
@@ -889,3 +889,42 @@ def delete_unused_nodes(self, outputs_name):
         """
         related_nodes = self.extract_sub_graph_nodes(outputs_name)
         self.set_nodes(related_nodes)
+
+
+class GraphBuilder(object):
```

Comment:
not sure whether we need this. seems not much benefit having another graph constructor. 

If you feel existing graph has defect, it's fine to modify that.